Person.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+#include <string>
+#include <iostream>
+
+class Person
+{
+    //enum gender{male, female};
+    
+    protected:
+        std::string firstName_;
+        std::string lastName_;
+        int64_t personalID_;
+        bool gender_;
+        std::string adress_;
```

Comment:
Lets think about it, maybe it will be better to use address as struct, or we should add it directly to person class (city, street...)